src/internal/events.ts

```diff
@@ -12,6 +13,12 @@ export const AccountCreatedEventStruct = object({
      * New account object.
      */
     account: KeyringAccountStruct,
+
+    /**
+     * Instructs MetaMask to display the add account confirmation dialog in the UI.
+     * Not guaranteed to be honored by MetaMask client.
```

Comment:
Was looking for something in JSDoc to "emphasis" this line a bit more, but couldn't find anything good...

Reply:
Addressed here [3154b12](https://github.com/MetaMask/keyring-api/pull/307/commits/3154b127d5732f7882108d47d542b28a3263a77a) :)